language-server/src/dotty/tools/languageserver/DottyLanguageServer.scala

```diff
@@ -297,7 +297,7 @@ class DottyLanguageServer extends LanguageServer
     implicit val ctx = driver.currentCtx
 
     val pos = sourcePosition(driver, uri, params.getPosition)
-    val tp = Interactive.enclosingType(driver.openedTrees(uri), pos)
+    val tp = Interactive.enclosingType(driver.openedTrees.getOrElse(uri, Nil), pos)
```

Comment:
There's many calls to `openedTrees(uri)` in this file, so instead of fixing it here I would change the definition of `openedTrees`.